examples/cosmos/ics20/denomTrace.qnt

```diff
@@ -67,6 +67,21 @@ module denomTrace {
       { path: [hopDestination].concat(denomTrace.path), baseDenom: denomTrace.baseDenom }
     }
   }
+
+  /// `isProperChannelSetup` is true iff channel names are unique by chain
+  pure def isProperChannelSetup(setup: str -> str -> str): bool =
```

Comment:
I was unable to infer the meaning of the `setup` parameter without reading the code.
I think the comment here should refer to and explain what's encoded in the `str -> str -> str` map.